modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -5,9 +5,34 @@ endif::[]
 
 If you encounter errors, refer to the following instructions for recovery.
 
-== Cannot Access the API Management Page to Disconnect Portal
-From the API Management, Settings page, you can disconnect your portal. If you cannot access the Settings page, you can disconnect the portal using the following Curl commands to unlink your Anypoint and Salesforce organization. You can run the commands in a terminal or using Postman and other applications.
+== Cannot See Images in the Builder Preview That Were Uploaded When Creating the Portal
+In the Builder preview, you cannot see the images that were uploaded when creating the portal because the shared user is not a member of any libraries.
+
+To add the shared user as a member of the asset library:
 
+. Log in to the Salesforce organization as a System Administrator.
+. Click the *App Launcher*, enter *files* in the search, and select *Files*. 
```

Comment:
Global: As per our style guide, we should use "click" for buttons and "select" for user actions in the UI (See "select", "choose", and "click" at https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference).

Reply:
Thanks, Cris. The app launcher is a button/icon.